book/website/communication/social-media.md

```diff
@@ -20,40 +20,40 @@ Although, maintaining a social media account successfully can be time-consuming.
 (cm-social-media-platforms)=
 ## Social Media Platforms
 
-You could use academic-based social media sites such as ResearchGate or Academia.edu or more general ones such as Twitter, Facebook, Instagram, Youtube and Linkedin.
+You could use academic-based social media sites such as ResearchGate or Academia.edu or more general ones such as X (formerly Twitter), Facebook, Instagram, Youtube and LinkedIn.
 
 More than 20 million researchers (figure from spring 2021) have signed up to ResearchGate and it can therefore offer a great opportunity for sharing papers, connecting with colleagues through the messaging feature and can be used to advertise projects.
-There are similar benefits to using Academia.edu or Mendeley, although they are not used as much as Researchgate.
+There are similar benefits to using Academia.edu or Mendeley, although they are not used as much as ResearchGate.
 
 A regular criticism of these sites is the spamming of your email with constant updates that are hard to turn off.
 
-More general social media sites are very popular with academic audiences and a recent Nature survey {cite:ps}`Noorden2014social` highlighted that researchers mostly use Twitter and Linkedin for professional use rather than Facebook.
+More general social media sites are very popular with academic audiences and a Nature survey in 2014 {cite:ps}`Noorden2014social` highlighted that researchers mostly use X and LinkedIn for professional use rather than Facebook.
 
-This survey also showed that Twitter had a wider range of uses compared to the other social media platforms, such as following discussions, posting work content, discovering peers, discovering recommended papers and commenting on others research.
+This survey also showed that X had a wider range of uses compared to the other social media platforms, such as following discussions, posting work content, discovering peers, discovering recommended papers and commenting on others research.
 It has become the communication channel of choice for many researchers and is opening up research discussions at a much earlier stage than previously seen.
 
-Twitter can be used to build up a personal academic profile, for use throughout a research project and can also be used by organisations to promote their work and message.
-It can make research more visible and understandable to wider communities by allowing non-scientists to find new research instantaneously.
+X can be used to build up a personal academic profile, for use throughout a research project and can also be used by organisations to promote their work and message.
 Scientists can also be contacted directly by the public, which enables a more approachable level of communication {cite:ps}`Kelesidou2021scicomm`.
+However recent changes to the platform have also alienated some academics and in the longer term this may precipitate a more widespread move by researchers away from X to more open platforms {cite:ps}`VidalValero2023twitter`.
```

Comment:
```suggestion
However recent changes to the platform have also alienated some academics and in the longer term this may result in a more widespread move by researchers away from X to more open platforms {cite:ps}`VidalValero2023twitter`.
```

Suggesting slightly clearer language to help with translations and non-fluent English speakers